components/dashboard/src/Login.tsx

```diff
@@ -209,7 +209,7 @@ export function Login() {
                 </div>
                 <div className="flex-none mx-auto h-20 text-center">
                     <span className="text-gray-400">
-                        By signing in, you agree to our <a className="gp-link hover:text-gray-600" target="gitpod-terms" href="https://www.gitpod.io/terms/">terms of service</a>.
+                        By signing in, you agree to our <a className="gp-link hover:text-gray-600" target="gitpod-terms" href="https://www.gitpod.io/terms/">terms of service</a> and <a className="gp-link hover:text-gray-600" target="gitpod-privacy" href="https://www.gitpod.io/privacy/">privacy policy</a>.
```

Comment:
**question**: We are using a similar notice about terms when installing Gitpod. Does it make sense to also include the privacy policy link there, too?

https://github.com/gitpod-io/gitpod/blob/f56a9f21ceefd8c170da255adeb8bb79f876ad37/components/dashboard/src/Setup.tsx#L48

Cc @corneliusludmann because of a [relevant discussion](https://gitpod.slack.com/archives/C01KLC56NP7/p1641547245016600).

Cc @MrSimonEmms because of #7732 and in case we could drop the privacy link from the telemetry settings in the admin dashboard added in https://github.com/gitpod-io/gitpod/pull/7732#pullrequestreview-863474737.

Cc @lucasvaltl @gitpod-io/engineering-self-hosted